xml/System.Text.Json.Serialization.Metadata/JsonPropertyInfo.xml

```diff
@@ -94,11 +91,9 @@
 
 ## Remarks
 
-When resolving metadata via <xref:System.Text.Json.Serialization.Metadata.DefaultJsonTypeInfoResolver>, this property
-             will be populated with the underlying <xref:System.Reflection.MemberInfo> of the serialized property or field.
+When resolving metadata via <xref:System.Text.Json.Serialization.Metadata.DefaultJsonTypeInfoResolver>, this property will be populated with the underlying <xref:System.Reflection.MemberInfo> of the serialized property or field.
```

Comment:
Apropos, this has been updated in .NET 9 so that it works with the source generator as well:
```suggestion
When resolving metadata via the built-in resolvers, this property will be populated with the underlying <xref:System.Reflection.MemberInfo> of the serialized property or field.
```